npm_and_yarn/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG PNPM_VERSION=8.11.0
 ARG YARN_VERSION=3.7.0
 
 # See https://github.com/nodesource/distributions#installation-instructions
-ARG NODEJS_VERSION=18.x
+ARG NODEJS_VERSION=20.x
 
 # Check for updates at https://github.com/npm/cli/releases
```

Comment:
Not upgrading the NPM version together is due to this issue 👉 https://github.com/npm/cli/issues/6742, see more 👉 https://github.com/dependabot/dependabot-core/pull/7811, cc @yeikel.

<img width="772" alt="image" src="https://github.com/dependabot/dependabot-core/assets/4016742/3599ef5a-76f3-4df2-9e3f-7220b2b68663">